### DISCUSSION TITLE
How should I fix this error

### DISCUSSION THREAD

**Discussion by ChenShiTong-CN** [PMEAL/OpenPNM#2881] [Q&A]
<img width="538" alt="image" src="https://github.com/PMEAL/OpenPNM/assets/73372468/da2a7c5d-49ea-480c-aa39-300337f2d6c4">

**Comment by ma-sadeghi** [comment]
Hi,
To be able to help you, your question needs to be well defined:

1. Come up with a descriptive title
2. Describe the issue in your own words
3. Produce and show a minimum working example (MWE*)

*An MWE is a short script that reproduces the issue you're having. It's NOT necessarily the script you've been working on, but a small subset of it. You need to spend some time for producing a good quality MWE as it helps developers help you more efficiently.

**A screenshot is not an MWE.
